### PR TITLE
Gets rid of unused properties

### PR DESCRIPTION
**Pull request by carolwong492**
# Ticket Link

[Ticket](https://uhawaii.atlassian.net/browse/GROUPINGS-735)

# List of squashed commits
- Re-adds emptying table

# Test Checklist

- [x] Unit Tests Passed:
- [x] Jasmine Tests Passed:
- [x] General Visual Inspection:

### CODE REVIEW

**Review by mhodgesatuh — APPROVED** [review]
I don't see any issues

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -23,75 +23,32 @@ cas.send.renew=false
...
 groupings.api.include=:include
 groupings.api.owners=:owners
-groupings.api.assign_type_group=group
-groupings.api.assign_type_immediate_membership=imm_mem
-groupings.api.subject_attribute_name_uuid=uid
-groupings.api.operation_assign_attribute=assign_attr
-groupings.api.operation_remove_attribute=remove_attr
-groupings.api.operation_replace_values=replace_values
-groupings.api.privilege_opt_out=optout
-groupings.api.privilege_opt_in=optin
```
Comment (JorWo) [inline src/main/resources/application.properties:58]: These two lines are still needed in GroupAttributeServiceImpl.java

Comment (devgav) [reply src/main/resources/application.properties:58]: I believe the isTrio is needed in there as well. Good catch

**Comment by devgav** [comment]
@carolwong492 i believe that intelliJ has trouble distinguishing unused variables in our application.properties file. Because of this you likely need to do a global search for a lot of these variables which I personally have had some trouble with in the past using intelliJ, instead I use VSCode since the search feature is better.

File: src/main/resources/application.properties (hunk 1 above)
Comment (carolwong492) [reply src/main/resources/application.properties:58]: Are you talking about the UI or the API? This pull request is for the UI, not the API. I don't think that file exists on the UI side.

Comment (devgav) [reply src/main/resources/application.properties:58]: > This pull request is for the UI, not the API. I don't think that file exists on the UI side.

oops, sorry 2about that @carolwong49 seems like there was a mix up on the branch that this pr was going into. Well at least now you will know to not remove those from the API haha.

**Review by devgav — APPROVED** [review]

**Pull request merged**